tuf/testutils/swizzler.go

```diff
@@ -532,3 +537,141 @@ func (m *MetadataSwizzler) UpdateTimestampHash() error {
 	}
 	return m.MetadataCache.SetMeta(data.CanonicalTimestampRole, metaBytes)
 }
+
+// MutateRoot takes a function that mutates the root metadata - once done, it
+// serializes the root again
+func (m *MetadataSwizzler) MutateRoot(mutate func(*data.Root)) error {
+	signedThing, err := signedFromStore(m.MetadataCache, data.CanonicalRootRole)
+	if err != nil {
+		return err
+	}
+
+	var root data.Root
+	if err := json.Unmarshal(signedThing.Signed, &root); err != nil {
```

Comment:
Actually I can probably just grab the bytes from the store and directly unmarshal it to a `data.SignedRoot` object, if that would be more clear, since in this case we don't need to go through the signed stage